strawberryfields/decompositions.py

```diff
@@ -291,6 +291,73 @@ def clements_phase_end(V, tol=1e-11):
     return (new_tlist, new_diags)
 
 
+def rectangular_symmetric(V, tol=1e-11):
+    r"""Rectangular decomposition of a unitary into symmetric beamsplitters.
```

Comment:
... into an array of symmetric beamsplitters?

Reply:
Implemented this in commit c230be7.